blockbase/functions.php

```diff
@@ -102,8 +102,13 @@ function blockbase_scripts() {
 	require get_template_directory() . '/inc/social-navigation.php';
 }
 
-require get_template_directory() . '/inc/fonts/custom-fonts.php';
 
+/**
+ * Do not use the custom fonts provided by Blockbase if Jetpack fonts module is active
+ */
+if ( ! class_exists( 'Jetpack' ) || ! Jetpack::is_module_active( 'google-fonts' ) ) {
```

Comment:
@simison is this a safe way to check?

Reply:
I think so! but let's @Automattic/jetpack-crew for 2nd opinion

Reply:
Clever doing module check instead of "is dotcom" check 👍

Reply:
Yes, it should work. The module will always be considered as "active" on WordPress.com simple, and is force-activated on WoA, so you'll end up using the module on both platforms at all times.

Reply:
I tested this PR in .com simple sites and the file didn't get loaded. 👍

Reply:
Is there any chance that the font module will become part of a standalone plugin at some point? If so, it might be better to do it like this:
```suggestion
if ( ! class_exists( \Automattic\Jetpack\Modules::class ) || ! ( new \Automattic\Jetpack\Modules() )->is_active( 'google-fonts' ) ) {
```
(feel free to put a `use Automattic\Jetpack\Modules;` at the top of the file instead of using a fully-qualified class name here, of course).